products/views.py

```diff
@@ -1,3 +1,67 @@
-from django.shortcuts import render
+import json
+
+from django.http        import JsonResponse
+from django.views       import View
+
+from products.models    import Category, Product, ThemeProduct, Theme, Material, ProductImage, ProductOption, Size, Color
+from users.models       import Like
+
+class DetailView(View):
+    
+    def get(self, request, product_id):
+        try:
+            product_data           = Product.objects.get(id=product_id)
```

Comment:
product가 좀 더 올바른 변수명이라고 생각됩니다.
```suggestion
            product                = Product.objects.get(id=product_id)
```

Reply:
변경하였습니다~!